uo_sys_framework/AOLimitModule/cfgFunctions/common/fn_DisplayAOLimitWarning.sqf

```diff
@@ -3,62 +3,62 @@
 params ["_display"];
 
 [{!(displayNull isEqualto _this)}, {
-	disableserialization;
+    disableserialization;
 
-	_this params ["_display"];
-	private _timecheckInit = CBA_missionTime;
-	private _endTime = (_timecheckInit + (missionNamespace getVariable ["UO_FW_AOL_TimeLeft", 30]));
-	private _nextBeep = _endTime - 10;
+    _this params ["_display"];
+    private _timecheckInit = CBA_missionTime;
```

Comment:
`_timecheckInit` is redundant. It is just a copy of a variable and next line can be written as `(CBA_missionTime + (missionNamespace getVariable ["UO_FW_AOL_TimeLeft", 30]));`